src/greynirseq/utils/bpe/multiprocessing_bpe_encoder.py

```diff
@@ -104,11 +104,15 @@ def initializer(self):
 
     def encode(self, line):
         global bpe
+        if self.args.add_prefix_space:
+            line = " " + line
         ids = bpe.encode(line)
         return list(map(str, ids))
 
     def decode(self, tokens):
         global bpe
+        if self.args.add_prefix_space:
```

Comment:
I think this should be more defensively programmed.

Reply:
This is the same behavior as in tokenizers, and in fashion with --keep-empty